index.js

```diff
@@ -55,43 +105,24 @@ async function run() {
         oldAdminSha = image.newName.split(":").slice(-1)[0]
         image.newName = `gcr.io/cdssnc/notify/admin:${adminSha.slice(0,7)}`
       }
-      if(image.name == "api"){ 
+      if(image.name == "api"){
         oldApiSha= image.newName.split(":").slice(-1)[0]
         image.newName = `gcr.io/cdssnc/notify/api:${apiSha.slice(0,7)}`
       }
     });
 
     const newBlob = Base64.encode(YAML.stringify(fileContent))
 
-    if(newBlob !== data.content){
-
-      // Close old auto PRs
-      const {data: prs} = await octokit.pulls.list({
-        owner: 'cds-snc',
-        repo: 'notification-tf',
-        state: 'open'
-      });
-  
-      prs.forEach( async pr => {
-        if(pr.title.startsWith("[AUTO-PR]")) {
-          await octokit.pulls.update({
-            owner: 'cds-snc',
-            repo: 'notification-tf',
-            pull_number: pr.number,
-            state: "closed"
-          });
-          await octokit.git.deleteRef({
-            owner: 'cds-snc',
-            repo: 'notification-tf',
-            ref: `heads/${pr.head.ref}`
-          });
-        }
-      })
+   if(newBlob !== data.content){
+      closePRs()
 
       const adminMsgs = await getCommitMessages("notification-admin", oldAdminSha)
       const apiMsgs = await getCommitMessages("notification-api", oldApiSha)
-  
+
       let logs = `ADMIN: \n\n ${adminMsgs.join("\n")} \n\n API: \n\n ${apiMsgs.join("\n")}`
+      if(hasChangesInStaging()){
+        logs = `Warning! There was a merged Pull Requests in notification-staging-tf since last deploy in production! \n\n ${logs}`
+      }
```

Comment:
```suggestion
      if(hasChangesInStaging()){
        logs = `⚠️ There was a merged pull request in [notification-staging-tf](https://github.com/cds-snc/notification-staging-tf) since last deploy in production! \n\n ${logs}`
      }
```